radicle-cli/examples/rad-id.md

```diff
@@ -145,7 +145,7 @@ increased to `1`.
 Instead, let's accept the proposal:
 
 ```
-$ rad id accept de4102c1b9b9b83683d7d9ca80c79ffebd62ac83 --no-confirm
+$ rad id accept de4102c --no-confirm
```

Comment:
I think @cloudhead mentioned that we shouldn't shorten the output. Which I think makes sense because we can't determine how much of the SHA we can show to be unambiguous.

Reply:
In fact this is the input?

Reply:
Oof sorry, I misread 🤦‍♂️